.github/ISSUE_TEMPLATE/lts-bug.yml

```
@@ -0,0 +1,107 @@
+name: "\U0001F41C LTS bug report"
+description: Submit a bug report to help us improve an LTS version of Iroha (iroha2-lts branch)
+title: "[BUG] Change title"
+labels: [ "Bug", "iroha2" ]
+body:
+  - type: input
+    id: commit-hash
+    attributes:
+      label: GIT commit hash
+      description: |
+        What is the commit hash of your Iroha version?
+        Old versions may have more bugs.
+      placeholder: abc123
+    validations:
+      required: true
+  - type: textarea
+    id: mwe
+    attributes:
+      label: Minimum working example code
+      description: |
+        Please share a minimal code that allows reproducing a problem.
+        Please enable [syntax highlighting](https://help.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks#syntax-highlighting).
+      placeholder: |
+        ```rust
+        fn main() {
+        }
+        ```
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected behavior
+      description: What do you expect the result to be?
+      placeholder: I expect Iroha to run normally.
+    validations:
+      required: true
+  - type: textarea
+    id: actual-behavior
+    attributes:
+      label: Actual behavior
+      description: What is the result you get?
+      placeholder: |
+        I get an error message when running Iroha:
+
+        Example error #123 
+    validations:
+      required: true
+  - type: input
+    id: os-type
+    attributes:
+      label: Operating system
+      description: |
+        What is the operating system you used when you encountered the issue?
+      placeholder: Arch Linux
+    validations:
+      required: true
+  - type: input
+    id: libc-type-and-version
+    attributes:
+      label: LibC type and version
+      description: |
+        What is the type of LibC you're using (`GNU libc` or `musl libc`)?
+        What is its version?
+        Use the `ldd --version ldd` command to determine it.
+        Write `N/A` if this command is unavailable on your system.
```

```suggestion
        Please specify the type of LibC you're using (`GNU libc` or `musl libc`) and its version.
        Use the `ldd --version ldd` command to determine it. Write `N/A` if this command is unavailable on your system.
```

also a question: why specify "N/A" if that field is not a required one, which means it could be left empty if the contributor cannot for some reason specify libc and its version